src/Common/Commands.Common.Authentication/Factories/AuthenticationFactory.cs

```diff
@@ -26,7 +26,9 @@ namespace Microsoft.Azure.Commands.Common.Authentication.Factories
 {
     public class AuthenticationFactory : IAuthenticationFactory
     {
-        public const string CommonAdTenant = "Common", DefaultMSILoginUri = "http://localhost:50342/oauth2/token";
+        public const string CommonAdTenant = "Common", 
+            DefaultMSILoginUri = "http://169.254.169.254/metadata/identity/oauth2/token", 
```

Comment:
Why are we using this specific IP and is it never going to change or should we use a DNS name?

Reply:
Because it is the defined ip of the metadata service on every VM host.